doc/release/upcoming_changes/21001.performance.rst

```diff
@@ -0,0 +1,3 @@
+Faster reduction operators
+--------------------------
+Reduction operations like `numpy.sum`, `numpy.prod`, `numpy.add.reduce`, `numpy.logical_and.reduce` on contiguous integer-based arrays are now much faster.
```

Comment:
Would be nice to break the long line, but doesn't really matter.

Reply:
Fixed.